python/crecordio.go

```diff
@@ -10,11 +10,39 @@ import "C"
 import (
 	"log"
 	"os"
+	"sync"
```

Comment:
Not for this line directly.
But we may add a comment at the top to illustrate the requirements from the build tool, in case ppl may get confused by mixing .go and .py together here.

Reply:
Added